README.md

```diff
@@ -110,3 +110,50 @@ Future<dynamic> backgroundMessageHandler(Map<String, dynamic> message) async {
 
 ### iOS
 Make sure that you have a `NSPhotoLibraryUsageDescription` entry in your `Info.plist`.
+
+If you don't have any push notifications integrated in your application you need to:
+1. Enable push notification in you provisioning profile.
+2. Add Capability `Push Notifcations` in your mobile application.
+3. Request notifications permissions. This library allow you to request all (badge, alert, sound) at once. Just use `Intercom.requestIosNotificationPermissions()`. It will return Future<bool> if user allowed for requested permissions.
+4. Add this code to your AppDelegate (before return statement):
+
+Swift:
+```
+DispatchQueue.main.async {
+  application.registerForRemoteNotifications()
+}
+```
+obj-c:
+```
+dispatch_async(dispatch_get_main_queue(), ^{
+  [application registerForRemoteNotifications];
+});
+```
+If you have already working notifications in your application non of above is required.
```

Comment:
I think this sentence is better located before this section, so user can skip it (maybe even with a link to the next required section).

Also, maybe it could be like this:
```suggestion
If you have already implemented notifications in your project, you can skip this section.
```